### PR TITLE
Adding iwd7 and also disabling iwd for xcode 7 for now.

### PR DESCRIPTION
**Pull request by moizjv**
- currently iwd does not work with xcode7 [see] (https://github.com/ios-driver/ios-driver/pull/388)

### CODE REVIEW

File: lib/instruments.js [hunk 1]
```diff
@@ -389,6 +389,8 @@ Instruments.prototype.spawnInstruments = function () {
   } else if (xcodeVer === 6) {
     iwdPath = path.resolve(thirdpartyPath, "iwd6");
   } else if (xcodeVer === 7) {
+    // iwd currently does not work with xcode7, setting withoutDelay to false
```
Comment (jlipps) [inline lib/instruments.js]: LGTM, however i'd like to log something saying we're skipping due to ios7

Comment (moizjv) [inline lib/instruments.js]: ok let me add an `log.info` here

**Comment by moizjv** [comment]
@jlipps done!

File: lib/instruments.js [hunk 2]
```diff
@@ -389,6 +389,10 @@ Instruments.prototype.spawnInstruments = function () {
...
+    logger.info("On xcode 7.0, instruments-without-delay does not work, skipping" +
+                 "instruments-without-delay");
```
Comment (jlipps) [inline lib/instruments.js]: this line needs to be dedented 1 char

**Pull request merged**